src/structures/Widget.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const Base = require('./Base');
+const WidgetUser = require('./WidgetUser');
+const WidgetChannelManager = require('../managers/WidgetChannelManager');
+
+/**
+ * Represents the data about the guild any bot can preview, connected to the specified guild.
+ * @extends {Base}
+ */
+class Widget extends Base {
+  constructor(client, data, id) {
+    super(client);
+
+    if (!data) return;
+    this.id = id;
+    this._patch(data);
+
+    this.members = [];
```

Comment:
can probably be a MemberManager of sorts..

Reply:
The problem is this is not a real member, all of informations in it aren't true

Reply:
Id's are also fake id's for these members, but you could have it be a new member manager

Reply:
True, I need to find something else